foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java

```diff
@@ -151,7 +153,8 @@ private static void duplicateServiceCombConfigToCse(ConcurrentCompositeConfigura
   }
 
   public static DynamicWatchedConfiguration createConfigFromConfigCenter(Configuration localConfiguration) {
-    if (localConfiguration.getProperty(configCenterUrlKey) == null) {
+    if (localConfiguration.getProperty(configCenterUrlKey) == null &&
```

Comment:
why need another control flag?
different ConfigCenter use the same "configCenterUrlKey" is better?

if you must use another key, then i remember we support map a key to another.

Reply:
Currently, newly imported configuration source would not configured by cse.config.client.serviceUri, it could be apollo.config.serviceUri, so I think that a new control flag such as configEnable would be in need. Old control flag reserved beacause the compatibility for old apps was considered.

I'll check the mapping feature and discuss this later.

Reply:
Currently, I remove newly added control flag and use original cse.config.client.serverUri to control. Need some help about how map different key to make this more elegant. I appreciate everything you do for me @wujimin. thanks again.

Reply:
Improvement of this problem will be watched at https://issues.apache.org/jira/browse/SCB-210